Library/Homebrew/extend/os/mac/keg_relocate.rb

```diff
@@ -137,6 +137,15 @@ def find_dylib(bad_name)
     lib.find { |pn| break pn if pn.to_s.end_with?(suffix) }
   end
 
+  def relocate_build_prefix(old_cellar, new_prefix, new_cellar)
+    macho_files.each do |file|
+      next if !file.mach_o_executable? && !file.dylib? && !file.mach_o_bundle?
```

Comment:
Same comment for `macho_files`. (I also thought it was `mach_o_files`).

Reply:
This was a typo and caused it to fail CI. Since I'm switching to `Keg#binary_executable_or_library_files` anyway this will be gone.